fenix.json.example

```diff
@@ -22,9 +22,7 @@
     "feed_info": {
         "publisher_name": "Torsten Grote",
         "publisher_url": "https://transportr.grobox.de",
-        "version":  "0.1",
-        "start_date": "20160901",
-        "end_date": "20170831"
```

Comment:
We should start documentation somewhere anyway. A lot of things are in code but not documented. Probably the wiki here would be a good place.